### PR TITLE
Update OTP-UI April 17 2024

### PR DESCRIPTION
**Pull request by miles-grant-ibigroup**
**Description:**

**PR Checklist:**
- [ ] Does the code follow accessibility standards (WCAG 2.1 AA Compliant)?
- [ ] Are all languages supported (Internationalization/Localization)?
- [ ] Are appropriate Typescript types implemented?

### CODE REVIEW

File: lib/util/config-types.ts [hunk 1]
```diff
@@ -317,6 +317,8 @@ export interface TransitOperatorConfig extends TransitOperator {
 export interface RouteViewerConfig {
   /** Whether to hide the route linear shape inside a flex zone of that route. */
   hideRouteShapesWithinFlexZones?: boolean
+  /** Remove vehiles from the map if they haven't sent an update in a number of seconds */
```
Comment (amy-corson-ibigroup) [inline lib/util/config-types.ts:320]: codespell is slacking

**Review by amy-corson-ibigroup — APPROVED** [review]
Beat me to it again!!! Thank you

File: lib/util/config-types.ts (hunk 1 above)
Comment (miles-grant-ibigroup) [reply lib/util/config-types.ts:320]: for every job AI steals from us, we steal one from AI

File: yarn.lock [hunk 2]
```diff
@@ -2383,7 +2383,7 @@
   dependencies:
     "@octokit/openapi-types" "^10.0.0"
 
-"@opentripplanner/base-map@^3.0.16", "@opentripplanner/base-map@^3.0.17":
+"@opentripplanner/base-map@^3.0.16":
```
Comment (binh-dam-ibigroup) [inline yarn.lock:2386]: Can this be deduped?

**Review by binh-dam-ibigroup — APPROVED** [review]

**Pull request merged**